main.py

```diff
@@ -214,7 +213,7 @@ def kcidb_spool_notifications(event, context):
     # Spool notifications from subscriptions
     for notification in kcidb.monitor.match(OO_CLIENT.query(pattern_set)):
         if not SELECTED_SUBSCRIPTIONS or \
-           notification.subscription in SELECTED_SUBSCRIPTIONS:
+                notification.subscription in SELECTED_SUBSCRIPTIONS:
```

Comment:
Is there a need to change this indent and to remove the empty line above?

Reply:
PEP 8 in pycharm was complaining of the indent 
But I have reverted it back

Reply:
Ah, good. Does CI catch this? If not, then we need to fix CI.

However, in any case, we need to change this in a separate commit, since it has nothing to do with the PR's change. At this point you would need to organize your commits not by just how you made them locally, but by how they will go into the repo. So far it seems there should be one commit for whitespace/formatting fixes with explanations what required them, and one commit for the main change.